kimchi/src/prover.rs

```diff
@@ -208,9 +208,9 @@ where
             .ok_or(ProverError::NoRoomForZkInWitness)?;
 
         let zero_knowledge_limit = zk_rows_strict_lower_bound(num_chunks);
-        if (index.cs.zk_rows as usize) < zero_knowledge_limit {
+        if (index.cs.zk_rows as usize) <= zero_knowledge_limit {
```

Comment:
And therefore, here, before if `zk_rows` was equals to 4, it would not have raised the error, which was not the expected behavior. Because 5 is the minimum number of `zk_rows`.